src/components/ui/text.stories.tsx

```
@@ -0,0 +1,153 @@
+    children: "Form Label Text",
+  },
+};
+
+export const Muted: Story = {
+  args: {
+    variant: "muted",
+    children: "This is muted text for less important content.",
+  },
+};
+
+export const Error: Story = {
```

Biome (2.3.13)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)